src/haxelib/client/Main.hx

```diff
@@ -178,15 +180,16 @@ class Main {
 		apiVersion : "3.0",
 		noSsl : false
 	};
-	static var IS_WINDOWS = (Sys.systemName() == "Windows");
+	static final IS_WINDOWS = (Sys.systemName() == "Windows");
+
+	final commands : List<{ name : String, doc : String, f : Void -> Void, net : Bool, cat : CommandCategory }>;
+	final isHaxelibRun : Bool;
+	final alreadyUpdatedVcsDependencies:Map<String,String> = new Map<String,String>();
 
-	var argcur : Int;
-	var args : Array<String>;
-	var commands : List<{ name : String, doc : String, f : Void -> Void, net : Bool, cat : CommandCategory }>;
-	var siteUrl : String;
-	var site : SiteProxy;
-	var isHaxelibRun : Bool;
-	var alreadyUpdatedVcsDependencies:Map<String,String> = new Map<String,String>();
+	var argcur:Int;
```

Comment:
The only diff here is the loss of spaces, but we're not being consistent in that regard (e.g. `final commands` just above).

Reply:
Reverted for the unchanged members